src/app/public/modules/repeater/repeater.component.spec.ts

```diff
@@ -686,4 +723,243 @@ describe('Repeater item component', () => {
+      fixture.detectChanges();
+      tick();
+      fixture.detectChanges();
+      let items = el.querySelectorAll('sky-repeater-item');
+      const itemToTest = items[1];
+      const itemDragHandle = el.querySelectorAll('.sky-repeater-item-grab-handle')[1];
+      SkyAppTestUtility.fireDomEvent(itemDragHandle, 'keydown', { keyboardEventInit: { key: ' ' } });
+      fixture.detectChanges();
+      SkyAppTestUtility.fireDomEvent(itemDragHandle, 'blur');
+      fixture.detectChanges();
+      SkyAppTestUtility.fireDomEvent(itemDragHandle, 'keydown', { keyboardEventInit: { key: 'arrowUp' } });
+      fixture.detectChanges();
+      SkyAppTestUtility.fireDomEvent(itemDragHandle, 'keydown', { keyboardEventInit: { key: ' ' } });
+      fixture.detectChanges();
+      expect(el.querySelectorAll('sky-repeater-item')[0]).not.toBe(itemToTest);
+      expect(el.querySelectorAll('sky-repeater-item')[1]).toBe(itemToTest);
+    }));
+
+    it('should turn off reordering when escape is hit', fakeAsync(() => {
+      fixture.detectChanges();
+      tick();
+      fixture.detectChanges();
+      let items = el.querySelectorAll('sky-repeater-item');
+      const itemToTest = items[1];
+      const itemDragHandle = el.querySelectorAll('.sky-repeater-item-grab-handle')[1];
+      SkyAppTestUtility.fireDomEvent(itemDragHandle, 'keydown', { keyboardEventInit: { key: ' ' } });
+      fixture.detectChanges();
+      SkyAppTestUtility.fireDomEvent(itemDragHandle, 'keydown', { keyboardEventInit: { key: 'Escape' } });
+      fixture.detectChanges();
+      SkyAppTestUtility.fireDomEvent(itemDragHandle, 'keydown', { keyboardEventInit: { key: 'arrowUp' } });
+      fixture.detectChanges();
+      expect(el.querySelectorAll('sky-repeater-item')[0]).not.toBe(itemToTest);
+      expect(el.querySelectorAll('sky-repeater-item')[1]).toBe(itemToTest);
+    }));
+
+    it('should revert any reordering up when escape is hit', fakeAsync(() => {
+      fixture.detectChanges();
+      tick();
+      fixture.detectChanges();
+      let items = el.querySelectorAll('sky-repeater-item');
+      const itemToTest = items[1];
+      const itemDragHandle = el.querySelectorAll('.sky-repeater-item-grab-handle')[1];
+      SkyAppTestUtility.fireDomEvent(itemDragHandle, 'keydown', { keyboardEventInit: { key: ' ' } });
+      fixture.detectChanges();
+      SkyAppTestUtility.fireDomEvent(itemDragHandle, 'keydown', { keyboardEventInit: { key: 'arrowUp' } });
+      fixture.detectChanges();
+      SkyAppTestUtility.fireDomEvent(itemDragHandle, 'keydown', { keyboardEventInit: { key: 'Escape' } });
+      fixture.detectChanges();
+      expect(el.querySelectorAll('sky-repeater-item')[0]).not.toBe(itemToTest);
+      expect(el.querySelectorAll('sky-repeater-item')[1]).toBe(itemToTest);
+    }));
+
+    it('should revert any reordering down when escape is hit', fakeAsync(() => {
+      fixture.detectChanges();
+      tick();
+      fixture.detectChanges();
```

Comment:
Looks like we might be able to move these to a `beforeEach()`?

Reply:
Good call. Done